src/scss/components/_card-citation.scss

```diff
@@ -0,0 +1,39 @@
+.card-citation {
+  &__header {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    font-size: 12px;
+    justify-content: flex-start;
+  }
+
+  &__title {
+    color: get-color(blackNeutral);
+    display: flex;
+    flex-wrap: wrap;
+    font-size: inherit;
+    font-weight: get-weight(normal);
+    margin-bottom: 0;
+
+    strong {
+      font-weight: get-weight(bold);
```

Comment:
`strong` is already `bold`: https://github.com/getninjas/gaiden/blob/master/src/scss/generic/_reset.scss#L11-L13

Reply:
@eduardojmatos  Nice!